target/linux/ramips/mt76x8/base-files/etc/init.d/bootcount

```diff
@@ -9,6 +9,9 @@ boot() {
 			echo -e "bootcount\nbootchanged\n" | /usr/sbin/fw_setenv -s -
 		;;
 	xiaomi,mi-router-4c|\
+	xiaomi,miwifi-3c)
```

Comment:
Err, why separate the -nano?

Reply:
Is not needed in breed bootloader. Fixed 2c753b0.